src/Hangfire.Mongo/Signal/Mongo/MongoSignalManager.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Threading;
+using Hangfire.Logging;
+using Hangfire.Server;
+
+namespace Hangfire.Mongo.Signal.Mongo
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class MongoSignalManager : IBackgroundProcess, IServerComponent
+    {
+        private static readonly ILog Logger = LogProvider.For<ExpirationManager>();
+
+        private readonly MongoStorage _storage;
+        private readonly IPersistentSignal _signal;
+
+        /// <summary>
+        /// Constructs expiration manager with one hour checking interval
+        /// </summary>
+        /// <param name="storage">MongoDB storage</param>
+        public MongoSignalManager(MongoStorage storage)
```

Comment:
Why not inject the HangfireDbContext.Signal from MongoStorage instead of the entire MongoStorage object?

Reply:
Sounds like a good plan...